spring-cloud-skipper-server-core/pom.xml

```diff
@@ -236,7 +247,9 @@
 					<plugin>
 						<groupId>org.apache.maven.plugins</groupId>
 						<artifactId>maven-surefire-plugin</artifactId>
+						<version>2.22.2</version>
```

Comment:
Where was the version coming from before? Is there a way to set the `2.22.2` once somewhere?

Reply:
Moved version to parent pluginManagement